src/main/java/io/vertx/redis/client/RedisException.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2018 Red Hat, Inc.
+ * <p>
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ * <p>
+ * The Eclipse Public License is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * <p>
+ * The Apache License v2.0 is available at
+ * http://www.opensource.org/licenses/apache2.0.php
+ * <p>
+ * You may elect to redistribute this code under either of these licenses.
+ */
+package io.vertx.redis.client;
+
+/**
+ * Maps a redis error message to a JVM Exception type.
+ * Redis Errors have no trace so this throwable will not contain a stacktrace.
+ *
+ * @author Paulo Lopes
+ */
+public final class RedisException extends Throwable {
```

Comment:
why extends Throwable ? you can get the no stack trace property with regular exceptions too